test/user.js

```diff
@@ -157,5 +158,55 @@ describe('UserService',function(){
 			_save.restore();
 			done();
 		});
-	})
-});
+	});
+
+	describe('deleteUser', function() {
+		var toDeleteUser;
+		var invalidToDeleteUser;
+		var _destroy;
+		before(function(done) {
+			toDeleteUser = User.forge({ id: 1, email: 'to@delete.com'});
+			invalidToDeleteUser = User.forge({ id: 2, email: 'cant@delete.com'});
+			invalidToDeleteUser.setPassword('password123')
+				.then(function(updatedUser) {
+					updatedUser.related('roles').add({ role: utils.roles.ATTENDEE});
+
+					_destroy = sinon.spy(User.prototype,'destroy');
+
+					done();
+				});
+		});
+		beforeEach(function (done) {
+			tracker.install();
+			done();
+		});
+		it('deletes a user with no role', function(done) {
+			tracker.on('query', function (query) {
+				query.response([]);
+			});
+
+			var deleteValid = UserService.deleteUser(toDeleteUser);
+			deleteValid
+				.then(function () {
+					assert(_destroy.calledOnce, "User destroy not called");
+
+					done();
+				})
+				.catch(function (err) {
+					done(err);
+				});
+		});
+		it('throws an exception when deleting a user with a role', function(done) {
+			expect(() => UserService.deleteUser(invalidToDeleteUser)).to.throw(errors.InvalidParameterError);
```

Comment:
`.to.eventually.throw` failed saying that the exception was thrown. This was the only one of three things I tried that worked